src/css/index.scss

```diff
@@ -1,2 +1,167 @@
 @import './global';
 @import './information';
+
+$basic-blue: #bcc8dc;
+$basic-grey: #edeff5;
+$dark-red: #d93347;
+$dark-blue: #314a74;
+
+.HOME {
+  position: fixed;
+  left: 70px;
+  top: 68px;
+  color: white;
+  font-size: 20px;
+  font-weight: bold;
+}
+.HOME:hover {
+  color: $basic-blue;
+}
+
+header {
+  height: 200px;
+}
+
+section ul {
+  height: 440px;
+  width: 948px;
+  position: relative;
+  left: 220px;
+  top: 230px;
+  align-items: flex-start;
+  align-content: flex-start;
+}
+.wrapper-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  font-weight: bold;
+  text-align: center;
+  position: relative;
+  left: 360px;
+  top: 130px;
+}
+
+.wrapper-box > * {
+  padding: 20px;
+  flex-grow: 0;
+  vertical-align: middle;
+}
+
+//Skills Name and Title
+.dark_red_font {
+  color: $dark-red;
+  position: relative;
+  top: 40%;
+}
+
+.dark_bule_font {
+  color: $dark-blue;
+  position: relative;
+  top: 40%;
+}
+
+.smaller-font {
+  position: relative;
+  top: 40%;
+  font-size: 60%;
+}
+
+///////////// Skills
+.skills-section {
+  height: 730px;
+  z-index: 0;
+  background: url('../css/背景.png') fixed no-repeat;
+}
+
+.skills-header-public {
+  position: relative;
+  font-size: 30px;
+  top: 78px;
+  font-weight: bold;
+}
+
+.skills-header {
+  @extend .skills-header-public;
+  color: white;
+  left: 360px;
+}
+
+.skills-header::after {
+  content: '';
+  display: block;
+  width: 50px;
+  border: 3px solid red;
+  margin: 28px 0;
+}
+
+.skill-Excel {
+  flex-grow: 1;
+  height: 145px;
+  background-color: $basic-grey;
+  font-size: 90%;
+}
+
+.skill-Word {
+  flex-grow: 1;
+  height: 145px;
+  background-color: $basic-blue;
+  font-size: 90%;
+}
+
+.skill-Adobe {
+  flex-grow: 2;
+  height: 215px;
+  font-size: 130%;
```

Comment:
Use `rem` unit for a relative font size. Please refer to: https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97#targetText=rem,the%20browser%20(usually%2016px).